### ISSUE TITLE
完善建议

### ISSUE THREAD

**Issue by feisher** [yadav-rahul/TastyToast#20]
建议修改吐司为静态吐司  在TastyToast类中修改 代码如下：

 static Toast toast = null; //Toast的对象！
    public static Toast makeText(Context context, String msg, int length, int type) {
        if (toast==null) {
            toast= new Toast(context);
        }
替换原此位置代码即可

**Comment by yadav-rahul** [comment]
@feisher LGTM. Are you interested in implementing this part?

**Comment by feisher** [comment]
@yadav-rahul Yes, I am very interested, but I am sorry that my Github is not  use good, I have to modify their own local library, very few online modification,

**Comment by yadav-rahul** [comment]
OK.

**Comment by yadav-rahul** [comment]
@feisher Are you working on this?

**Comment by yadav-rahul** [comment]
@feisher If you are working on this one then please update your progress regarding it.

**Comment by nullkk** [comment]
@yadav-rahul Thank you for this libs I found clicking the same button multiple times Will appear many times toast, He was revised , toast Singleton mode.

 
 
<img width="851" alt="wx20180126-152629 2x" src="https://user-images.githubusercontent.com/20369464/35429088-6faa98fe-02ad-11e8-9ad3-889d43784d11.png">

**Comment by nullkk** [comment]
@feisher  老哥 多次点击 还是会多次出现 能不能只出现一次,他这个 toast搞成了单例, 但是view 不能只绘制一次吧 ?

**Comment by feisher** [comment]
@nullkk改写后的效果是这样，是为了 保证实时显示的是最新的toast提示，肯定需要二次绘制的，不然怎么更新内容 
 ![mark](http://ovddupnnd.bkt.clouddn.com/blog/180127/IJgiLDHkKl.gif)

**Comment by nullkk** [comment]
@feisher 我把layout的view也写成单列了. 每次调用判断是否为空,不为空直接showview.这样只加载一次view, 但是会有问题 比如我点了error 类型 set10个文字 在点info类型set 3个文字 view是只加载了一次 但是文字大小都设置不一样.这个view就不会刷新..

**Comment by ichapter** [comment]
Hi,Thank you very much for open source this library,There is a small suggestion,Is it possible to modify the background color of toast?Because some pages themselves are white background, it is not obvious，thank.

**Comment by gzehua** [comment]
Android高版本中关闭notification权限,还能显示吗?

**Comment by feisher** [comment]
@gzehua toast和通知权限无关，完全不受影响的
